lib/eventful.ex

```diff
@@ -8,6 +8,7 @@ defmodule Eventful do
     binary_id = Keyword.get(options, :binary_id)
     {parent_relation, parent_module} = Keyword.get(options, :parent)
     {actor_relation, actor_module} = Keyword.get(options, :actor)
+    event_relation = Keyword.get(options, :event_relation) || "#{parent_relation}_events"
```

Comment:
Can you change the option name to `:table_name`

Reply:
@zacksiri 
Yes this is done.